spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerTemplate.java

```diff
@@ -31,13 +32,17 @@
 import com.google.cloud.spanner.Key;
 import com.google.cloud.spanner.KeySet;
 import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Options.QueryOption;
+import com.google.cloud.spanner.Options.ReadOption;
 import com.google.cloud.spanner.ReadContext;
 import com.google.cloud.spanner.ReadOnlyTransaction;
 import com.google.cloud.spanner.ResultSet;
 import com.google.cloud.spanner.Statement;
 import com.google.cloud.spanner.TimestampBound;
 import com.google.cloud.spanner.TransactionContext;
 import com.google.cloud.spanner.TransactionRunner.TransactionCallable;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
```

Comment:
spring-jcl is included in the spring-cloud-gcp project, we should use their Logger implementation instead of apache commons logging.

Reply:
Ah, ignore this, it's fine - the spring-jcl is actually implementing the apache commons logging interfaces (https://docs.spring.io/spring/docs/current/javadoc-api/org/apache/commons/logging/package-summary.html)